extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -35,20 +34,25 @@ public function index() {
 		wfProfileIn( __METHOD__ );
 
 		// Since this "Community_App" article won't be found, we need to manually say it's okay so that it's not a 404.
-		$this->response->setCode( self::RESPONSE_OK );
+		$this->response->setCode( static::RESPONSE_OK );
 
 		// Pull in the app-configuration (has data for all apps)
 		$appConfig = [];
 		$memcKey = wfMemcKey( static::$CACHE_KEY, static::$CACHE_KEY_VERSION );
```

Comment:
In MAIN-7821 branch I've replaced this whole first batch of app-config stuff with a call to getAllAppConfigs() since there's no reason to have this in here twice (I should have replaced this as soon as I created getAllAppConfigs()).